test/spec/modules/inskinBidAdapter_spec.js

```diff
@@ -12,7 +12,8 @@ const REQUEST = {
     'bidder': 'inskin',
     'params': {
       'networkId': 9874,
-      'siteId': 730181
+      'siteId': 730181,
+      'publisherId': 123456
```

Comment:
Could you include this param in your markdown file

Reply:
@ncolletti This has been done, thanks!